Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -117,11 +138,34 @@ public CosmosClientOptions()
         /// Gets and sets the preferred regions for geo-replicated database accounts in the Azure Cosmos DB service. 
         /// </summary>
         /// <remarks>
-        /// When this property is specified, the SDK will use the region list in the provided order to define the endpoint failover order.
-        /// This configuration is an alternative to <see cref="ApplicationRegion"/>, either one can be set but not both.
+        /// <para>
+        /// During the SDK initialization the account information, including the available regions, is obtained from the <see cref="CosmosClient.Endpoint"/>.
+        /// The SDK will use the value of <see cref="ApplicationPreferredRegions"/> to populate the preferred list with the account available regions that intersect with its value.
```

Comment:
```suggestion
        /// The SDK will use the value of <see cref="ApplicationPreferredRegions"/> to populate the preferred list with the account available regions.
```

Do we need to mention the intersection part? I think it is unnecessary because you are already explaining it below.

Reply:
IMO It is important to highlight that it's an intersection, this should help customers understand that if the account has 10 regions, and they only put 3 regions on the preferred list, then only those 3 regions will be used (unless they go all offline in which case we would pick from the others). 

The below explanation is for the edge case where the account is in 10 regions, I specify 3 regions but 2 of those 3 do not exist in the account, which means, only 1 would be used on the preferred list.

Reply:
Got it, makes sense!